app/src/main/java/com/fastaccess/ui/modules/search/SearchUserActivity.kt

```diff
@@ -0,0 +1,95 @@
+package com.fastaccess.ui.modules.search
+
+import android.content.Context
+import android.content.Intent
+import android.os.Bundle
+import android.support.v7.app.AppCompatActivity
+import android.text.Editable
+import butterknife.ButterKnife
+import butterknife.OnClick
+import butterknife.OnEditorAction
+import butterknife.OnTextChanged
+import com.fastaccess.R
+import com.fastaccess.helper.AnimHelper
+import com.fastaccess.helper.InputHelper
+import com.fastaccess.helper.Logger
+import com.fastaccess.ui.modules.search.repos.SearchReposFragment
+import kotlinx.android.synthetic.main.activity_search_user.*
+
+class SearchUserActivity : AppCompatActivity() {
+
+    companion object {
+
+        @JvmStatic
+        val USERNAME = "username"
+        @JvmStatic
+        val SEARCH_TERM = "search"
+
+        @JvmStatic
+        fun getIntent(context: Context, username: String, searchTerm: String?): Intent {
+            val intent = Intent(context, SearchUserActivity::class.java)
+            intent.putExtra(USERNAME, username)
+            intent.putExtra(SEARCH_TERM, searchTerm)
+            return intent
+        }
+    }
+
+    private var username = ""
+    private var searchTerm = ""
+    private var isFork = true
+    private lateinit var searchReposFragment: SearchReposFragment
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_search_user)
+        ButterKnife.bind(this)
+
+        searchReposFragment = SearchReposFragment.newInstance()
+        supportFragmentManager.beginTransaction()
```

Comment:
that's a crash on Orientation change. 

you should only `commit` when `savedInstanceState == null`